### DISCUSSION TITLE
Stuck at Moving through time level - (branches section)

### DISCUSSION THREAD

**Discussion by vuld0** [git-learning-game/oh-my-git#151] [General]
I am stuck at moving through time level. The challenge seems unclear to me. It says that we have to `git checkout` at a commit and edit the files. Though, I have completed, but the challenge doesn't complete.

**Comment by tunafishman** [comment]
just get the state of the piggy bank and the little sister's coin counts back to what it was.

**Comment by Matias-Ramos** [comment]
Checkout to last branch. Modify piggy bank so it has 10 coins again and commit. Yeah, the prompt is a bit unclear.
